guide/miniscript-wallet-simple.md

```diff
@@ -0,0 +1,366 @@
+
+If the recovery path is enabled, our application uses smart defaults to propose a suitable key scheme, based on the wallet. Since we are creating a 2-of-3 wallet, the application will suggest a 1-of-3 recovery path. This recommendation will be different for other wallet types. 
+
+It also suggests that the recovery path will be unlocked after 6 months, if the funds have not been spent. However, users can always choose a different timeframe that better suits their situation.
+
+In addition, our application also provides users with the option to set reminders at different times before the recovery path unlocks.
+
+{% include image-gallery.html pages = page.images_creation-recovery-path %}
+
+Note that the scope of this feature is quite narrow because it is aimed at an audience with basic knowledge of bitcoin. You might want to offer more customization options, based on your own target audience.
+
+#### Save for later
+
+As you can see, creating a multi-key wallet involves a lot of steps in the wallet application as well as on the corresponding signing devices. This can take quite a while, depending on the chosen key scheme. It also requires that users have all extended public keys ready and available beforehand.  
+
+It is likely that not all users will be able or willing to go through the entire process in one setting. This is why our application offers a feature called "Save for later". It allows users to pause the wallet creation process at any time and pick it up later. The application saves the current state of the wallet locally on the device, until it is actually created on-chain.
+
+{% include image-gallery.html pages = page.images_save-for-later %}
+
+#### Wallet backup 
+
+Enabling the recovery path is the final step before the wallet is created on-chain. Once it has been finalized and can be used, our application prompts the user to download the wallet backup file. 
+
+This is important, because users will need the wallet configuration file or the [wallet descriptor](https://bitcoin.design/guide/glossary/#output-script-descriptor), if they should ever need to recover their wallet. The wallet descriptor functions like a map that contains all the information necessary for a wallet application to restore the wallet. Without this map, the wallet cannot be recovered even if you have backups of all private keys. 
+
+{% include image-gallery.html pages = page.images_backup %}
+
+The backup file does not contain any private key data. This means that it is ok to store it digitally in a password manager, for example. However, since it contains all extended public keys, anybody who finds this file can potentially monitor all transactions in real time.
+
+Because they might not want to backup the wallet right away, users will find the backup file and other important information on the wallet settings page at any time. 
+
+
+### Co-signer onboarding
```

Comment:
Seems like the idea here is that Bob sets up everything, and then gives Alice one of the hardware wallets for signing. That means Alice is not involved in the wallet creation process and fully trusts Bob not to keep a backup copy of the hardware wallet she will be using, right?

It might be a better scenario, where Bob and Alice have separate hardware wallets and collaborate during the wallet creation process. So Bob never gets to touch Alice's keys/backup, and vice-versa.